blog/2023-12-21-alert-configuration-manager/index.md

```diff
@@ -0,0 +1,55 @@
+
+---
+slug: introducing-netdata-alerts-configuration-manager
+title: "Introducing Netdata’s Alerts Configuration Manager"
+description: |
+  Discover the power of Netdata's new Alerts Configuration Manager, a user-friendly UI wizard for creating and managing alerts directly from your Netdata Dashboard.
+authors: satya
+tags: [netdata, alerts-configuration-manager, UI-dashboard, alert-management]
+keywords: [netdata, alerts configuration, UI wizard, dashboard, alert management]
+image: ./img/alert-config-manager.png
+---
+
+
+slug: netdata-cost-transparency-unveiling-the-true-cost-of-monitoring
```

Comment:
all this and below seems to be a duplicate from previous blogpost

Reply:
This was me checking what was wrong.. ;)